Funky-Chickens/workFORCEmanagement
# Cait delete

added functionality in controllers to delete a computer and delete a training program. added pug views for training programs and delete button for computer and training program.

to test
make sure you have run "npm install method-override"
navigate to the training programs to view a list of all training programs
navigate to a single training program (such as "/training/2"), hit the delete button, see that the record was deleted from the list it returns to you.
next navigate to a single computer (such as "/computers/2"), hit the delete button, see that the record was deleted from the list it returns you to.
